c/check.py

```diff
@@ -279,8 +364,40 @@ def check_file_has_no_windows_line_ending(self):
         if any('\r' in line for line in self.lines):
             self.error("Windows line endings")
 
+    def check_no_unknown_property(self):
+        # Check each file name part immediately after a _true- or _false- for a valid property
+        for part in (self.filename.split("_true-")[1:] + self.filename.split("_false-")[1:]):
+            if not any(part.startswith(prop) for prop in PROPERTIES):
+                self.error("has unknown property " + part)
```

Comment:
Is there already a check that there exists a property at all?

Reply:
Yes, here: https://github.com/sosy-lab/sv-benchmarks/blob/089b0bf3d1a7424d56ff445b4305fb3f3e01e924/c/check.py#L243-L245